deploy.sh

```diff
@@ -1,4 +1,14 @@
 #!/usr/bin/env bash
 set -e
-pip3 install -r src/requirements.txt
+git fetch --all
+latest=$(git tag -l 'v*' | sort | tail -n 1)
```

Comment:
A suggestion here:

```
versions=$(git tag --list 'v*' | sort -n | tail -n5 | head -n4 | tr '\n' ' ')
latest=$(git tag -l 'v*' | sort -n | tail -n 1)
echo -n "Which tag do you want to deploy? ...$versions[$latest] "
```

in any case, `sort -n` is useful if we hit double-digits on a tag part as it should sort better (11 after 2)

Reply:
The above will print

```
Which tag do you want to deploy? ...v0.7.1 v0.7.2 v0.7.3 v0.7.4 [v0.7.5] 
```